src/main/kotlin/com/terraformation/backend/accelerator/event/ParticipantProjectSpeciesAddedToProjectNotificationDueEvent.kt

```diff
@@ -0,0 +1,12 @@
+package com.terraformation.backend.accelerator.event
+
+import com.terraformation.backend.db.accelerator.DeliverableId
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.SpeciesId
+
+/** Published when a participant project species is added to a project */
```

Comment:
Comment should probably mention something about this being a deferred, throttled event rather than one that's published immediately when a participant project species is added to a project.